PLEP-0007.rst

```diff
@@ -0,0 +1,262 @@
+|   ``diagnostics`` sub-packages, then the feature should be included one of   |
+|   respective sub-packages.  For example, a synthetic diagnostic should be    |
+|   included in the ``plasmapy.diagnostics`` sub-package.                      |
++------------------------------------------------------------------------------+
+| .. _tests:                                                                   |
+|                                                                              |
+| **tests**                                                                    |
++------------------------------------------------------------------------------+
+| :directory: ``./plasmapy/tests/``                                            |
+| :access: ``plasmapy.tests``                                                  |
+| :PLEP:                                                                       |
+| :scope:                                                                      |
+|   A collection of tests for top-level modules (i.e. functions and classes    |
+|   defined in top-level ``.py`` files).                                       |
+|                                                                              |
+|   Utility's associated with running and developing tests (e.g.               |
+|   ``pytest_helpers``) should also be included here over ``plasmapy.utils``.  |
++------------------------------------------------------------------------------+
+| .. _utils:                                                                   |
+|                                                                              |
+| **utils**                                                                    |
++------------------------------------------------------------------------------+
+| :directory: ``./plasmapy/utils/``                                            |
+| :access: ``plasmapy.utils``                                                  |
+| :PLEP:                                                                       |
+| :scope:                                                                      |
+|   A collection of "utility" functions and classes to help us write           |
+|   (what we try to think of as) clean, readable, and informative code.        |
+|                                                                              |
+|   This collection does not provide any physics tools, instead it is          |
+|   focused on providing package development tools.                            |
+|                                                                              |
+|   .. note:: Utilities focused on running and developing tests should be      |
+|             placed in ``plasmapy.tests`` instead.                            |
++------------------------------------------------------------------------------+
+
+Implementation
+==============
+
+Implementing this PLEP requires refactoring (renaming of moving) of existing
```

Comment:
```suggestion
Implementing this PLEP requires creating new and renaming or moving existing
```

Reply:
(at the very least, `of` in the parentheses should be changed to `or`)

Reply:
ok...re-worded